bosh-stemcell/spec/os_image/ubuntu_xenial_spec.rb

```diff
@@ -195,6 +195,17 @@
       it { should be_file }
       it { should be_executable }
     end
+
+    context 'zfs' do
+      %w[
+        /lib/modules/*/kernel/zfs/
+        /usr/src/linux-headers-*/zfs
+      ].each do |folder|
+        describe file(folder) do
```

Comment:
Does `file` automatically do glob'ing, or is literally looking for a `*` in the path?